build-system/test-configs/dep-check-config.js

```diff
@@ -169,6 +169,9 @@ exports.rules = [
   {
     filesMatching: '{src,extensions}/**/*.js',
     mustNotDependOn: '3p/**/*.js',
+    allowlist: [
+      'src/inabox/inabox-iframe-messaging-client.js->3p/iframe-messaging-client.js',
```

Comment:
curious about the size of the intersection-observer-polyfill extension.

Reply:
This particular change is actually completely unrelated to my change :) We are already using 3p/ in the src/inabox/. Somehow my change just triggered this dep check.

But the InOb polyfill itself is ~4K. We maybe adding an extra 0.5K or so to plug it all in.